packages/dtd-catalog-exporter/src/config/config.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  FileManagerConfig,
+  LoggerConfig,
+  ReadModelDbConfig,
+  S3Config,
+} from "pagopa-interop-commons";
+import { z } from "zod";
+
+const DtdCatalogExporterConfig = ReadModelDbConfig.and(FileManagerConfig)
+  .and(S3Config)
```

Comment:
For the S3 Bucket, to have the same env var name as the one from the reports repo, you could remove this `S3Config` and the `S3_BUCKET` env var from the `.env` file, then add a more specific `DTD_CATALOG_STORAGE_BUCKET`